src/views/Home/components/CLWCreditsDaysLeft/index.js

```diff
@@ -187,14 +187,23 @@ export default class CLWCreditsDaysLeft extends Component {
       <Card>
         <CardContent>
           <Grid container direction="row" alignItems="center">
-            <Grid item xs={7} align="left">
+            <Grid item xs={4} align="left">
+              <Button
+                variant="contained"
+                style={style.button}
+                onClick={(event) => (window.location.href = '/attended')}
+              >
+                ATTENDED EVENTS
+              </Button>
+            </Grid>
+            <Grid item xs={4} align="center">
               <CardHeader title={this.state.currSessionDescription} />
             </Grid>
-            <Grid item xs={5} align="right">
+            <Grid item xs={4} align="right">
               <Button
                 variant="contained"
                 style={style.button}
-                onClick={(event) => (window.location.href = '/events?CLW')}
+                onClick={(event) => (window.location.href = '/events?CLW%20Credits')}
```

Comment:
Just like above, we can make the routing much smoother and quicker using the `Link` component from `react-router-dom`:
```suggestion
                component={Link}
                to="/events?CLW%20Credits
```